etc/db_schema.xml

```diff
@@ -0,0 +1,21 @@
+<schema xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Setup/Declaration/Schema/etc/schema.xsd">
+    <table name="rp_auction_items">
+        <column xsi:type="int" name="entity_id" unsigned="true" nullable="false"
+                identity="true" comment="Auction Item ID"/>
+        <column xsi:type="varchar" name="item_name" nullable="false"
+                length="255" comment="Auction Item Name"/>
+        <column xsi:type="varchar" name="item_sku" nullable="true"
+                length="255" comment="Auction Item Sku"/>
+        <column xsi:type="decimal" name="item_base_auction_price" unsigned="true"
```

Comment:
It would be good to specify the precision of the price column. Looks like you can use the "scale" attribute: https://devdocs.magento.com/guides/v2.4/extension-dev-guide/declarative-schema/db-schema.html